test/is-low-quality-comment.ts

```diff
@@ -25,4 +25,5 @@ test('isLowQualityComment', t => {
 	t.false(isLowQualityComment('Same here. <some useful information>'));
 	t.false(isLowQualityComment('Same here, please update, thanks'));
 	t.false(isLowQualityComment('Same here! Please update, thank you.'));
+	t.false(isLowQualityComment('[here](https://www.example.com)'));
```

Comment:
Added a test case to cover https://github.com/refined-github/refined-github/issues/5257#issue-1091939044:

> Also we should ensure we don't hide comments that have links because `[here](##)` could still be useful.